src/cli/expand-patterns.js

```diff
@@ -65,9 +66,13 @@ async function* expandPatternsInternal(context) {
       continue;
     }
 
-    const stat = await statSafe(absolutePath);
+    const stat = await lstatSafe(absolutePath);
     if (stat) {
-      if (stat.isFile()) {
+      if (stat.isSymbolicLink()) {
+        yield {
+          error: `Explicitly specified symbolic link was ignored: ${pattern}`,
```

Comment:
"was ignored" seems not right, since we exited. 


`Explicitly specified ${pattern} is a symbolic link.` ?

Reply:
Fixed.